R/uploadPackage.R

```diff
@@ -40,9 +40,12 @@ uploadPackageUI <- function(id) {
             )
         ),
         actionLink(NS(id, "upload_format"), "View Sample Dataset")
-      )
-    ),
+      ),
+     ),
     
+     fluidRow(
+        uiOutput(NS(id, "rem_pkg_div"))
+    ),
```

Comment:
Currently this fluid row has awkward spacing under the upload package selectize input. This is because of the introJS. You can easily circumnavigate this issue by putting the uiOutput in the same column as the package upload text input.
![image](https://user-images.githubusercontent.com/85879620/213550447-ab8020ac-168c-49bd-b96e-7c014a39621a.png)
VS
![image](https://user-images.githubusercontent.com/85879620/213550730-4a3a11cd-96f9-4f62-b8c6-40b51c9039a3.png)

Reply:
Done, now it looks like the second layout.